pallet/dex/src/mock.rs

```diff
@@ -183,6 +184,7 @@ impl ExtBuilder {
 		<pallet_dex::GenesisConfig as frame_support::traits::GenesisBuild<Test>>::assimilate_storage(
       &pallet_dex::GenesisConfig {
         asset_ids: vec![999, 888, 777],
+        native_asset_id: 888,
```

Comment:
We need to make same decimal points for asset ggx. (e.g 18 decimals instead of 10)